danger_code_ws/1.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
```

Comment:
Line exceeds the recommended maximum length of 80 characters. Consider breaking this line into multiple lines for better readability.